runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/Process/Process.thunks.ts

```diff
@@ -52,7 +52,7 @@ export const fetchGuestDemoProcess = createAsyncThunk<IProcess>(
         .then((response) => response.data),
 );
 
-export const updateProcess = ApiTenantResource.patch<IProcess, IProcess>('processes/getByName', PROCESSES_PATH);
+export const updateProcess = ApiTenantResource.patch<IProcess, IProcess>('processes/update', PROCESSES_PATH);
 
 export const partialUpdateProcess = createAsyncThunk<IProcess, IProcess, { rejectValue: any }>(
```

Comment:
what is the difference between these functions?

Reply:
I am not sure, it should be done in separate task.